environments/ci/docker-compose.yaml

```diff
@@ -0,0 +1,15 @@
+version: "3.8"
+
+services:
+  core:
+    image: localhost:5000/core:${DOCKER_IMAGE_TAG_CI}
+    build:
+      args:
+        - BASE_IMAGE=ubuntu:20.04
+        - PYTHON_VERSION=3.8
+        - RUN_POETRY_INSTALL_AT_BUILD_TIME=true
+      context: ../../
+      dockerfile: environments/Dockerfile
+    tty: true
```

Comment:
```suggestion
```
I believe we don't  need tty in CI process.

Reply:
https://github.com/cvpaperchallenge/Ascender/actions/runs/3312637449
Unfortunately, `docker-compose exec -T core mv ../.venv .` was failed without `tty=true`.